sql/hive/pom.xml

```diff
@@ -238,36 +238,13 @@
              <configuration>
                <sources>
                  <source>v${hive.version.short}/src/main/scala</source>
-                 <source>${project.build.directory/generated-sources/antlr</source>
+                 <source>${project.build.directory}/generated-sources/antlr</source>
```

Comment:
It didn't seem to be affecting the build, though. I'm actually wondering if this line is needed - the parser seems to be part of the catalyst module, not this one.